sdk/identity/azcertificatemanager/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 0.1.0 (2023-01-17)
+
+* Template package validating release pipeline
```

Comment:
This will need to be updated, see https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/CHANGELOG.md#010-2020-01-10 for an example.